src/Arcus.Templates.WebApi/Program.cs

```diff
@@ -22,21 +53,24 @@ public class Program
 #if Serilog
         #warning Make sure that the appsettings.json is updated with your Azure Application Insights instrumentation key.
         private const string ApplicationInsightsInstrumentationKeyName = "Telemetry:ApplicationInsights:InstrumentationKey";
-
+        
+#endif
+#if SharedAccessKeyAuth
+        private const string SharedAccessKeyHeaderName = "X-API-Key";
+        
 #endif
         public static int Main(string[] args)
         {
```

Comment:
Should we migrate here to the same minimized version of Program.cs like the one that is generated by VS.NET 2022 when starting a new project ?
(One with top level statements, so no namespace declaration, no class specification, no Main method .... ) ?

It might become too complex though.

Reply:
Hmm, we should be careful of introducing new syntactic sugar to our common templates. Might be a good idea to stay with something that's common ground.
People can always change it.

Reply:
Well, it's just to be more aligned with how the VS.NET project templates for web apps look like.
I think this new syntax can be good for simple things; in our case, we do already have a lot of extra methods etc....   I agree to keep it this way for now.

Reply:
We can still move to the new syntax later, when people are maybe more accustomed to it.